sensorlib/src/noise_models/GaussianNoiseModel.py

```diff
@@ -68,3 +69,55 @@ def apply_list_inclusion_noise(self, object_list):
         if not self.__config["stages"]["list_inclusion_noise"]:
             return object_list
         return object_list[0:np.random.randint(0, len(object_list) + 1)]
+
+    def apply_position_covariance(self, object_list):
+        if not self.__config["stages"]["position_noise"]:
+            return object_list
+
+        # Apply position noise to the object_list
+        noise_std = self.__position_std
```

Comment:
This variable seems unnecessary. We can use `self.__position_std` directly in the `for` loop. Also, we should remove the double leading underscores.